tracer/test/snapshots/Security.AspNetCoreBare.__expectedStatusCode=500_url=_bad-param=[$slice].verified.txt

```diff
@@ -20,10 +20,12 @@ System.Exception: boom!
 at Samples.Security.AspNetCoreBare.Controllers.BadController.Get(),
       error.type: System.Exception,
       http.client_ip: 127.0.0.1,
+      http.endpoint: bad,
       http.method: GET,
       http.request.headers.host: localhost:00000,
       http.request.headers.user-agent: Mistake Not...,
       http.request.headers.x-forwarded: 86.242.244.246,
+      http.response.headers.content-length: 0,
```

Comment:
Is this correct? You send content when blocking don't you, so this shouldn't be 0?